### PR TITLE
Fix javadoc warnings and remove #if 0'd code

### PR DESCRIPTION
**Pull request by DanHeidinga**
Signed-off-by: Dan Heidinga <daniel_heidinga@ca.ibm.com>

### CODE REVIEW

File: jcl/src/java.base/share/classes/java/lang/ClassLoader.java [hunk 1]
```diff
@@ -1000,6 +1000,14 @@ static ClassLoader getClassLoader(Class<?> clz) {
 /*[ENDIF]*/
 
 /*[IF Sidecar19-SE]*/
+/**
+ * Return the Platform classloader.
+ *
+ * If a security manager exists and it does not
+ * allow access a SecurityException will be thrown.
```
Comment (pshipton) [inline jcl/src/java.base/share/classes/java/lang/ClassLoader.java]: There should be a `@throws` for this as well.

File: jcl/src/java.base/share/classes/java/lang/invoke/MethodHandle.java [hunk 2]
```diff
@@ -833,6 +836,14 @@ public MethodHandle asFixedArity() {
 	}
 /*[ENDIF]*/
 
+	/**
+	 * Bind the value as the first argument to the MethodHandle
+	 *
+	 * @param value the object to bind in
+	 * @return a new MethodHandle with the bound argument
+	 * @throws IllegalArgumentException if the value is not compatible with the first argument
+	 * throws ClassCastException if the value can't be cast to the type of the first argument
```
Comment (pshipton) [inline jcl/src/java.base/share/classes/java/lang/invoke/MethodHandle.java]: Missing `@`

**Comment by DanHeidinga** [comment]
Thanks @pshipton.  I've pushed a new commit that addresses the two issues

**Comment by pshipton** [comment]
Not sure why I didn't see it before, but the copyright check for KeyHashTable.c is failing.

**Comment by DanHeidinga** [comment]
Fixed now

**Pull request merged**